doc/install.rst

```diff
@@ -20,7 +20,7 @@ If you would like to save movies of time course data, it is necessary to include
 If you'd like to install the development version, you have two options. You can
 install straight from github::
 
-    pip install git+https://github.com/nipy/pysurfer.git
+    pip install -U https://api.github.com/repos/nipy/PySurfer/zipball/master
```

Comment:
If we're gonna put core numeric packages in ``install_requires`` we might want to add ``--no-deps`` here so people aren't upgrading lots of things they might not want to be.